src/components/CurrentEvent/CurrentEvent.tsx

```
@@ -0,0 +1,17 @@
+import { UniLayout } from '~components/Uni/Layout/UniLayout';
+import { eventCardLayoutTemplate } from '~features/events_list/components/EventsPanel/eventLayouts';
+
+export const CurrentEvent = () => {
+  const [{ data }] = useAtom(currentEventResourceAtom.v3atom);
+  if (!data) return null;
+
+  const currentEvent = { ...data, showEpisodesButton: false, active: true };
+
+  return (
+    <div className="knt-panel knt-current-event">
+      <UniLayout data={currentEvent} layout={eventCardLayoutTemplate}></UniLayout>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)